gdb/fbsd-kvm.c

```diff
@@ -331,6 +346,28 @@ fbsd_kvm_target_open (const char *args, int from_tty)
 	discard_cleanups(old_chain);
 	unpush_target(&fbsd_kvm_ops);
 
+	/* Relocate kernel objfile if needed. */
+	if (symfile_objfile &&
+	    (bfd_get_file_flags(symfile_objfile->obfd) &
+	      (EXEC_P | DYNAMIC)) != 0 &&
+	    ops->kern_disp != NULL) {
+		struct section_offsets *new_offsets;
+		int i;
+		CORE_ADDR displacement;
+
+		displacement = ops->kern_disp(target_gdbarch(),
```

Comment:
Why not pass 'nkvm' directly since it is already open?

Reply:
Because I would need to include kvm.h from kgdb.h (to get kvm_t) and I thought that could be a problem.
But in case it's ok, it really would be better to pass nkvm directly.